src/instructions/auf_dem_trockenen.md

```diff
@@ -28,27 +28,19 @@ For the spatial selection we also need the German border, which we found by sear
 
 <img src="images/auf_dem_trockenen_4.png" width="50%" height="auto" alt="Germany search">
 
-Next, for the link between the occurrence data and the average temperature, we search for the _Average Temperature_ dataset in the data catalogue.
+Next, for the link between the occurrence data and the average temperature, we search for the _Daily mean air temperature_ dataset in the data catalogue.
 
 <img src="images/auf_dem_trockenen_5.png" width="50%" height="auto" alt="Average temperature search">
 
-**Caution**: The _Average Temperature_ is a spatio-temporal dataset. Always check the spatial and temporal extent in the metadata.
+**Caution**: The _Daily mean air temperature_ is a spatio-temporal dataset. Always check the spatial and temporal extent in the metadata.
 
 <img src="images/auf_dem_trockenen_6.png" width="50%" height="auto" alt="Average temperature spatiotemporal extent">
 
-The _Average Temperature_ dataset covers the whole Earth and a time range from 1970/01/01 to 2000/12/31. To do this we need to change the time in the time menu at the top right.
+The *Daily mean air temperature* dataset covers Germany with a time range from 1951/01/01 to 2021/01/01 in a daily resolution. To see an effect on the data it is recommended to change the time in the time menu at the top right due to the high spatiotemporal resolution of the temperature data.
 
 <img src="images/auf_dem_trockenen_7.png" width="50%" height="auto" alt="Time menu">
 
-As the dataset does not look very attractive, we will change the colour palette of the raster data. This can be done by right-clicking on the layer and selecting _Edit Symbology_.
-
-<img src="images/auf_dem_trockenen_8.png" width="50%" height="auto" alt="Edit symbology button">
-
-In the symbology menu, scroll down to _Create colour table_, select a colour map such as _VIRIDIS_ or _MAGMA_, click the _Create colour table_ button and confirm with the _Apply button_ at the bottom of the symbology menu.
-
-<img src="images/auf_dem_trockenen_9.png" width="50%" height="auto" alt="Edit symbology menu">
-
-In order to link the occurrence data with the land use classification, it is also necessary to load the _Oekosystematlas_ by searching for it in the personal data catalogue. The personal data catalogue contains all datasets uploaded by the user as well as a section with _all datasets_, which also contains datasets not listed.
+In order to link the occurrence data with the land use classification, it is also necessary to load the _IÖR Landschaftsklassifikation_ by searching for it in the data catalogue.
```

Comment:
Again I would already call it a join here since the operator is also called join. It introduces less terms.